pubspec.yaml

```diff
@@ -13,8 +13,11 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
+  flutter_web_plugins:
+    sdk: flutter
   freezed_annotation: ^2.0.1
   json_annotation: ^4.8.0
+  web: ^1.1.1
```

Comment:
Since you depend on `package:web`, the [min. required SDK version is 3.4.0](https://github.com/dart-lang/web/blob/222164b5010cb0324d4132666f1d43750ffc6448/web/pubspec.yaml#L7), AND since the min. Dart version is 3.4.0, you can bump both `dart` and `flutter` versions at the top of this file ([corresponding Flutter version for this Dart version is 3.22.0](https://docs.flutter.dev/release/archive)).

```yaml
environment:
  sdk: ">=3.4.0 <4.0.0"
  flutter: ">=3.22.0"
```

Reply:
That's a very good point. Thanks for bringing it up!